### DISCUSSION TITLE
Introductory tour

### DISCUSSION THREAD

**Discussion by kj7rrv** [InfiniTimeOrg/InfiniTime#1820] [Ideas]
I was able to figure out how to use InfiniTime quite quickly, but a simple tour/tutorial for new users could be a helpful way to show users how to navigate the OS.

**Comment by VeryUsual** [comment]
There are tours of InfiniTime all over youtube/peertube. For example, this video https://yewtu.be/watch?v=CvT5D0kxoiQ shows all the features
